style.css

```diff
@@ -0,0 +1,31 @@
+div {
+  font-style: italic;
+  text-align: center;
+}
+@media (max-width:320px) {
+  div { font-size: .5em; color: blue } /* smartphones4 iPhone, portrait 480x320 phones */
+   }
+   
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/css17__/pullRequest?prid=5991486)